### DISCUSSION TITLE
Google Doc like comments

### DISCUSSION THREAD

**Discussion by Tejaswgupta** [toeverything/blocksuite#6700] [General]
Firstly, Kudos to the great work on the project!

I wanted mentions/comments functionality as available on Google Doc editor for my website. Upon some research I found https://tiptap.dev/product/comments and https://liveblocks.io/comments both of which are paid options.
Is BlocSuite planning on any similar feature in the near future? if not, could anyone possible give some pointers as to how we can implement a similar functionality as those but with BlockSuite and our stack:
- Next.js
- Supabase(including Supabase-realtime if relevant)

**Comment by Flrande** [comment]
We have plans to implement comments, and I have already written a POC. I believe we will have this feature soon.

https://github.com/toeverything/blocksuite/pull/6302
